### ISSUE TITLE
Errors and warnings are not being passed into console.warn and console.error as a string

### ISSUE THREAD

**Issue by petercpwong** [webpack-contrib/eslint-webpack-plugin#88]
- Operating System: Ubuntu 20.04
- Node Version:  v14.15
- webpack Version: 5.24.2
- eslint-webpack-plugin Version: 2.5.2

### Expected Behavior

Should emit errors and warnings in the console

### Actual Behavior

Warnings are emitted as an object in the console

![screenshot](https://user-images.githubusercontent.com/1303285/109799640-39e15600-7c57-11eb-89d6-a941ded84fab.png)

### Code

```
{
  module: {
    rules: [
      {
        test: /\.js$/,
        use: [
          {
            loader: 'babel-loader'
          }
        ]
      }
    ]
  },
  plugins: [new ESLintPlugin()]
}
```

**Comment by alexander-akait** [comment]
Bug in webpack-dev-server, already fixed, please wait release, sorry

**Comment by jasperkuperus** [comment]
Sorry if this is considered pushy, but when are you planning to release? Thanks in advance.

**Comment by undeadfrost** [comment]
This problem still exists now, which version needs to be updated?
